aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -690,7 +690,12 @@ class AlgoIterator {
         cudaGetLastError(); // clear CUDA error
       }
     }
-    TORCH_CHECK(false, "Unable to find a valid cuDNN algorithm to run convolution");
+    // This error doesn't necessarily mean the error originated from cudnn.
+    // In case where we have concurrent kernel failure outside of cudnn,
+    // we could also get here.
+    // One quick trick to rule out that is to set CUDA_LAUNCH_BLOCKING to
+    // avoid concurrent kernel execution.
```

Comment:
Nope, it's not concurrent kernel execution, it's asynchronous (so the error for an earlier kernel is reported later).

Reply:
This makes a lot more sense.
Thanks for the correction!